pywbem_mock/_cimrepository.py

```diff
@@ -2267,34 +2297,34 @@ def AssociatorNames(self, **params):
             if iname.host is None:
                 iname.host = self.host
 
-        return self._return_assoc_tuple(results)
+        return results
 
     ####################################################################
     #
     #   Associators server request
     #
     ####################################################################
 
-    def Associators(self, **params):
+    def Associators(self, ObjectName, AssocClass=None, ResultClass=None,
+                    Role=None, ResultRole=None, IncludeQualifiers=None,
+                    IncludeClassOrigin=None, PropertyList=None):
         """
-        Implements a mock WBEM server responder for
+        Implements a mock CIM repository responder for
             :meth:`~pywbem.WBEMConnection.Associators`
         """
 
         namespace = self.namespace
```

Comment:
See previous comment about `namespace = self.namespace`. Only this time, the namespace is (optionally) passed in ObjectName.